Bug/webapp/plugins services

rlebre opened the pull request:
Fixes #379 

The DicoogleClient API was updated. Thus, the request in the WebApp got outdated. This PR fixes the issues in Management > Services and Plugins > Plugins.

Enet4 commented on dicoogle/src/main/resources/webapp/js/stores/pluginStore.js:
```diff
@@ -2,40 +2,40 @@ import Reflux from "reflux";
     const self = this;
 
-    Dicoogle.request("GET", ["plugins", type], (error, data) => {
+    Dicoogle.request("GET", ["plugins", type]).end(function (error, data) {
```
You can keep using the arrow function syntax, so you can use `this` without having to create an outer `self`.

Enet4 commented on dicoogle/src/main/resources/webapp/js/stores/pluginStore.js:
```diff
@@ -2,40 +2,40 @@ import Reflux from "reflux";
     const self = this;
 
-    Dicoogle.request("POST", ["plugins", type, name, action], (error, data) => {
+    Dicoogle.request("POST", ["plugins", type, name, action]).end(function (error, data) {
```
Same here.

rlebre commented:
Indeed! 
Thank you for your suggestions. In fact, that way we may not use the outer self. I fixed that and also fixed the warning 
`Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of 'PluginsView'. See https://fb.me/react-warning-keys for more information.`
when opening the Services and Plugins tab.
